### ISSUE TITLE
Missing information ?

### ISSUE THREAD

**Issue by popujol** [MicrosoftDocs/dynamics-365-unified-operations-public#3770]
Hello, 

Is it normal that  client id created in ERP are 6f7d0213-62b1-43a8-b7f4-ff2bb8b7b452 and 2e49aa60-1bd3-43b6-8ab6-03ada3d9f08b and then in CRM we are asked to create application users and one of them hasn't got the same ID 00000015-0000-0000-c000-000000000000 and 2e49aa60-1bd3-43b6-8ab6-03ada3d9f08b.
Also isn't missing a piece ? after step 3. I cannot find the application users in CRM.

Thank you

Pierre-Olivier
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: df8e9efc-9258-3876-e05f-20d35c907274
* Version Independent ID: 4b5041ec-4a48-0565-e47a-b972e65a6d60
* Content: [System requirements and prerequisites - Finance & Operations | Dynamics 365](https://docs.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/data-entities/dual-write/requirements-and-prerequisites)
* Content Source: [articles/fin-ops-core/dev-itpro/data-entities/dual-write/requirements-and-prerequisites.md](https://github.com/MicrosoftDocs/Dynamics-365-Unified-Operations-Public/blob/main/articles/fin-ops-core/dev-itpro/data-entities/dual-write/requirements-and-prerequisites.md)
* Service: **dynamics-365-cross-app**
* GitHub Login: @NHelgren
* Microsoft Alias: **nhelgren**

**Comment by NHelgren** [comment]
@paulliew  can you please review the auth information in this document?

**Comment by kamaybac** [comment]
@popujol  , thank you for contacting us regarding this help topic. I have passed your comment on for consideration.

**Comment by kamaybac** [comment]
Bug 662876 created to track this issue.

**Comment by josaw1** [comment]
Closed as won't fix.
